examples/typescript/facoin/publish.json

```diff
@@ -0,0 +1,17 @@
830c436b6aff4b2d19fcdae7ee1658e04292482cc014f94e0a5a1640558436c205f1cd686c953c05aa708f0b61f0ab75708689d29746473a74703481889de934bc73bfda76da4603c6998d54c79ab76bacbb76689f6c54e7b069aa9ab16b25f19551497f403e92065c2603663a86336ab0330766228b9951cd046d28cc25e911e4a6d1a453807d713b46d4935ea6ca9d479ac01d571e0c67d0065db83606a0c497589c8533b6895c53d028c1e370ad713bed1e4a7cd684a188625619d8c1a650ddd5aefe2dea4657d923028a2d1a83fa79324325c44ba88dc1ec56872d7543b7477b2bab2b71fb3d496c470f12333e74be45a55d3db73a516c964e2eafc180e864981cf29ec650cff9c2bfaeb8de59dcc4270c3878ed8c8659aa3c3331368c3d589a6dca449089f36b82cf9d21e1f5ce8017b741866f5b364dce389a73f10242cb83302bcd2f8fa3551005b87215a95cf85eb54f9351b547533ef3a75dd751b54fae4738c4f6dda80729f168d523b738dba8a9f8369dea7f4f67a037dfaaead491f1b10e2dec6e336a2572d31800d7b501d82e270e50bf33925cbb6b2be210a54d8e3bd7fda0f7416ba59f9415a1ec1ac695bd49c4adfccc027dfc5581e2dea694933539cfa6d42e20ed23d7f221f0415ca184cbd5b2d0eaea8b828baf968fb896d1b066b94177702808f53d13a10f54c8efa95da945c0d44133c7d1e8097265cf66c85d8adabe110112ea1ffd16988e402aa480121c159c66ba67a612270b69903dc401e9470409f271e2768da2a2d6e0eea1128fbbba6303ec0a0d5c04240fb7804d9b0be9985a3bb56732e8056793ae5d0c22c35ffa43f1fccaa305779ba84ea63a4870e0ac2642d560e8a0b2a958c145fafb7c06b32df846d7bbdf7e3368d6cec3a0a433d0eb254f7a9a6780ce9ec2764b3d718dc73eaaf5c461aeb8ab727ff69d3b46ce010a0cf750f88ce0ac13f1076860a4aca7c8039dea315f68070d5e1cfb7235350adb76fc2a15e6003e0611f9b29d5e352a465be1d3d4336357b69e8d4a74d67473d0ef568dae919752f19f6b69b5834c0167d846c62ca572547a72bb81931bcc0d72245089eb1a79746de7200d8104a2f0521d0d5c37dcb957a24b9be06c30d8cf1362b10d75d5a46e839597f5f59c1ac308a53a775053d37de00f8dd27f467e59fdbed661cf5d16eeb675a947816a460937da226ab50fece8a61303f03c99afd8fe5d7c3b280bbda1a30df4211ce055e7e917357a3b8740a57d16b0b675054a1fded03f79f397343287df06e7436ba521b22fadc35d85f0a50d7fd5ddf7f1738c0f0bbcfcd6fd1eaac34ce3768903b8d8dd99501915655adf44b917ec22082b08cf8bba979d380cf673e6e67e3b6a9d697d173b8f8c23891f6715dcbb8eb3cb50c098801375ac319ab86b2129eb60d110602de5e2ab5544b9066dc0663e5866b1740ee6faf25c84de9a3b50b752bf2575a2a53e388bdbfc66eafed55971791991fc173927976864c25a2a252bfcca7396194e451fbf63beac0eb763d2bf914977bebfc3a68dc086fe4962aa3fefb0380a222ff0cb58fdcaddd3df69aeb0b2c48a758d20c4a3b5fb6b1764241b46f9e74ddb751a72b483737da79735813c9f5a2180b08652bf864e7a2c019e9bcd2999ba9138e435331daddd909556e28109e0543990c8061489a68c594c505edef6ba331badfcde67e3732e4ee277841a53ba887fc88d1838ebae6f23a1ca95ea875e0fe454c021ddd88c4e861f77175a543fb8498dcfa8ca0fcd44d427d471b0ec8cf3bfd423a94642e742d3b5dd6dd79472ee0514ae9c70d4c4b28ff1186bc57a9fee51056ef5c3cd8d9d9d98d51c933ac9fde614cff141626ed6a4dd990bb2656daa566fc7b4969fbfaa65dbb3655afb6aeb6fe010ca47f45191e000000000300000000000000000000000000000000000000000000000000000000000000010e4170746f734672616d65776f726b00000000000000000000000000000000000000000000000000000000000000010b4170746f735374646c696200000000000000000000000000000000000000000000000000000000000000010a4d6f76655374646c696200"
```

Comment:
Interesting, what is this file?  A format file for deploying packages?

Perhaps we want to make a github issue to autogenerate this from the CLI for easier deployment via TS and others.